src/server/game/gamemanager.js

```diff
@@ -1104,6 +1233,37 @@ const gamemanager = (function() {
     function isGameOver(game) { return game.gameConclusion !== false; }
 
     /**
+     * Returns *true* if the white in the provided game has a draw offer.
+     * @param {Game} game - The game
+     * @returns {boolean}
+     */
+    function hasWhiteDrawOffer(game) {
+        const isOffering = (game.whiteDrawOffer === 'offered')
+        return isOffering
+    }
+
+    /**
+     * Returns *true* if the black in the provided game has a draw offer.
+     * @param {Game} game - The game
+     * @returns {boolean}
+     */
+    function hasBlackDrawOffer(game) {
```

Comment:
This could probably be merged with hasWhiteDrawOffer() to form like a new function hasColorDrawOffer(game, color)